operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md

```diff
@@ -1,10 +1,26 @@
 ## Change NCN Image Root Password and SSH Keys
 
-Customize the NCN image by changing the root password or adding different ssh keys for the root account.
+Customize the NCN images by changing the root password or adding different ssh keys for the root account.
 This procedure shows this process being done on the PIT node during a first time installation of the CSM
 software.
 
-This process should be done for the "Kubernetes" image used by master and worker nodes and then repeated for the "ceph" image used by the utility storage nodes.
+There is some common preparation before making the Kuberentes image for master nodes and worker nodes and making the Ceph image for utility storage nodes and some common cleanup afterwards.
+
+
+***Note:*** This procedure can only be done before the PIT node is rebuilt to become a normal master node.
+
+### Common Preparation
+
+1. Prepare new ssh keys on the PIT node for the root account in advance.  The same key information will be added to both k8s-image and ceph-image.
+
+   Either replace the root public and private ssh keys with your own previously generated keys or generate a new pair with `ssh-keygen(1)`.  By default `ssh-keygen` will create an RSA key, but other types could be chosen and different filenames would need to be substituted in later steps.
```

Comment:
Remove extra space before second sentence.

Reply:
Fixed in next update.